ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__azure-4.17-nightly-x86.yaml

```diff
@@ -122,6 +133,90 @@ tests:
     - chain: openshift-qe-control-plane
     workflow: openshift-qe-installer-azure
   timeout: 5h0m0s
+- always_run: false
+  as: control-plane-multi-24nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: azure-arm64
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: aarch64
+      ADDITIONAL_WORKER_VM_TYPE: Standard_D4ps_v5
+      ADDITIONAL_WORKERS: "3"
+      BASE_DOMAIN: ci.azure.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "3"
+      COMPUTE_NODE_TYPE: Standard_D4s_v5
+      CONTROL_PLANE_INSTANCE_TYPE: Standard_D4s_v5
+      OCP_ARCH: amd64
+      OPENSHIFT_INFRA_NODE_INSTANCE_TYPE: Standard_E16s_v3
+      PROFILE_TYPE: reporting
+      SET_ENV_BY_PLATFORM: custom
+      WORKER_REPLICA_COUNT: "24"
+    test:
+    - ref: ipi-install-heterogeneous
+    - chain: workers-scale
+    - chain: openshift-qe-control-plane
+    workflow: openshift-qe-installer-azure
+  timeout: 5h0m0s
+- always_run: false
+  as: control-plane-multi-120nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: azure-perfscale
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: aarch64
+      ADDITIONAL_WORKER_VM_TYPE: Standard_D8ps_v5
```

Comment:
What's the rationale behind have `Standard_D8ps_v5` as opposed to `Standard_D4ps_v5` here in 120 nodes testing?

Reply:
I forgot to change it to `Standard_D4ps_v5` , I have changed it for 418 job